src/Http/Routing.Abstractions/src/Microsoft.AspNetCore.Routing.Abstractions.csproj

```diff
@@ -11,6 +11,7 @@ Microsoft.AspNetCore.Routing.RouteData</Description>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;routing</PackageTags>
     <IsShippingPackage>false</IsShippingPackage>
+    <ContainsManualReferenceAssemblyContent>true</ContainsManualReferenceAssemblyContent>
```

Comment:
Back out all changes to src/ projects; `$(ContainsManualReferenceAssemblyContent)` isn't needed.